src/main/java/com/spotinst/sdkjava/model/bl/gcp/LaunchSpecReq.java

```diff
@@ -0,0 +1,83 @@
+package com.spotinst.sdkjava.model.bl.gcp;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class LaunchSpecReq {
```

Comment:
1. Create a folder 'requests' under model folder. Move this class in the requests folder in correct hierarchy - model.requests.ocean.gke

Reply:
Resolved